src/grt/src/GlobalRouter.cpp

```diff
@@ -1956,7 +1957,8 @@ void GlobalRouter::readGuides(const char* file_name)
       odb::Rect rect(
           stoi(tokens[0]), stoi(tokens[1]), stoi(tokens[2]), stoi(tokens[3]));
       guides[net].push_back(std::make_pair(layer->getRoutingLevel(), rect));
-      boxToGlobalRouting(rect, layer->getRoutingLevel(), routes_[net]);
+      int layer_idx = layer->getRoutingLevel();
+      boxToGlobalRouting(rect, layer_idx, layer_idx, routes_[net]);
```

Comment:
Since the guides from a file won't have the via information to enable parasitic estimation we should give a message about that to the user.